functions/fisher.fish

```diff
@@ -2,6 +2,8 @@ function fisher --argument-names cmd --description "A plugin manager for Fish"
     set --query fisher_path || set --local fisher_path $__fish_config_dir
     set --local fisher_version 4.4.3
     set --local fish_plugins $__fish_config_dir/fish_plugins
+    test -e $fish_plugins && set --local file_plugins (string match --regex -- '^[^\s]+$' <$fish_plugins)
+    set --query _fisher_plugins || set --universal _fisher_plugins $file_plugins
```

Comment:
@Susensio What is this change about?

Reply:
This is a merge from #746 that I added into my personal fork, I did not know it would modify this PR

Reply:
Let me split this into multiple PRs so you can decide better

Reply:
The --no-config part is no longer needed. I thought that it was intended to avoid `status is-interactive` but it is more than that.
The subshell fisher starts to parallelize installation does not need to read config, as it seem (at the moment) to only write to temp folders and such. It wouldn't hurt to add this, but the improvement in speed is negligible so I think it's safer and simpler to **let it be as is**.